examples/lighting-app/efr32/README.md

```diff
@@ -80,84 +80,64 @@ Silicon Labs platform.
 
 *   Build the example application:
 
-          ```
```

Comment:
You probably want to keep the code blocks, so spellcheck is not confused.

Reply:
Good point! In that case, the backticks should be moved to the first column.

Reply:
I think one of the tests didn't pass earlier to a "prettier" issue. I resolved that, and now all tests are passing.